### ISSUE TITLE
🐛 Bug Report: scaffolder/next - markdown in section description no longer working

### ISSUE THREAD

**Issue by ScottGuymer** [backstage/backstage#17167]
### 📜 Description

When configuring a template with a section that includes a description you can currently include markdown in this description and things like links render correctly

```
...
spec:
  parameters:
    - title: Scaffolder Section
      description: |
        ***Warning: some warning to users.***
        [My Link](https://google.com)
      properties:
...
```

### 👍 Expected behavior

Text in the description should be rendered as markdown 

### 👎 Actual Behavior with Screenshots

In scaffolder/next markdown is no longer rendered correctly

### 👟 Reproduction steps

Use a template like

```
...
spec:
  parameters:
    - title: Scaffolder Section
      description: |
        ***Warning: some warning to users.***
        [My Link](https://google.com)
      properties:
...
```

and text is rendered explicitly and not processed as markdown

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by Pike** [comment]
This works for me on 1.13.0-next.1

**Comment by ScottGuymer** [comment]
Let me update to that version and test it out

**Comment by ScottGuymer** [comment]
Not stale

**Comment by benjdlambert** [comment]
@crivetechie @ScottGuymer  this seems to work for me. I wonder if you need to update to the latest release?

![Screenshot 2023-06-26 at 13 41 21](https://github.com/backstage/backstage/assets/3645856/686ea46e-771d-4e17-9893-959a7994053f)

**Comment by benjdlambert** [comment]
Granted the theme is a little off - but it works :tada:

**Comment by ScottGuymer** [comment]
It does seem like this was solved in 1.14 or 1.15 somewhere.

I have updated to 1.15 and it seems to now be working

![image](https://github.com/backstage/backstage/assets/956825/0bcdab1b-86d3-4e26-af68-4a2d5cb1545f)

**Comment by benjdlambert** [comment]
Nice - I actually think that it was https://github.com/backstage/backstage/pull/17952 that fixed this, and it also enables markdown in all descriptions on elements too.
